test/migrators/color/global.hrx

```diff
@@ -0,0 +1,39 @@
+<==> input/entrypoint.scss
+a {
+  b: red(gold);
+  c: green(gold);
+  d: blue(gold);
+  e: hue(gold);
+  f: saturation(gold);
+  g: lightness(gold);
+  h: adjust-hue(gold, 20deg);
+  i: saturate(gold, 10%);
+  j: desaturate(gold, 10%);
+  k: transparentize(gold, 0.1);
+  l: fade-out(gold, 0.1);
+  m: opacify(gold, 0.1);
+  n: fade-in(gold, 0.1);
+  o: lighten(gold, 10%);
+  p: darken(gold, 10%);
+}
+
+<==> output/entrypoint.scss
+@use "sass:color";
+
+a {
+  b: color.channel(gold, 'red', rgb);
```

Comment:
Can we have an explicit `$space: ` on these as well?

Reply:
Done